src/System.Runtime.InteropServices/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netstandard1.7-Windows_NT;
```

Comment:
Yeah, the .builds file here was already messed up (at least it looks like it to me), which  is what caused this one. I'll manually clean this up.